tests/test_case.py

```diff
@@ -6,11 +6,11 @@
 from faunadb.errors import NotFound
 from faunadb.objects import Ref
 
-_FAUNA_ROOT_KEY = environ["FAUNA_ROOT_KEY"]
+_FAUNA_ROOT_KEY = environ.get("FAUNA_ROOT_KEY", "secret")
```

Comment:
I don't understand. They'll have to set all this stuff anyway if their environment is different.